nifi-nar-bundles/nifi-language-translation-bundle/nifi-yandex-processors/pom.xml

```diff
@@ -53,7 +53,12 @@
             <artifactId>jersey-media-json-jackson</artifactId>
             <version>${jersey.version}</version>
         </dependency>
-        
+        <dependency>
+            <groupId>org.glassfish.jersey.inject</groupId>
+            <artifactId>jersey-hk2</artifactId>
```

Comment:
I think this is licensed under CDDL 1.1? If so it should be included in the NAR's NOTICE file along with the other Jersey stuff.

Reply:
I'll get the following line added to the NOTICE file:
`    (CDDL 1.1) (GPL2 w/ CPE) jersey-hk2 (org.glassfish.jersey.inject:jersey-hk2:jar:2.26 - https://jersey.github.io/)`

Reply:
@mattyb149 Added